Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -680,8 +794,12 @@ public function setCode($code)
             $this[$index] = new Token($token);
         }
 
+        $transformers = Transformers::create();
+        $transformers->transform($this);
```

Comment:
Yes, it should be ported into 1.4.
It won't affect our code, but may affect contributor's fixers that aren't in this repo.
Thanks for catching that !